src/lib/apiTypesV2/Registration.cpp

```diff
@@ -36,7 +36,7 @@ namespace ngsiv2
 *
```

Comment:
An entry regarding the changes included in this PR and citing the issue number should be added to CHANGES_NEXT_RELEASE_FILE

Reply:
Added in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/49b0c6df2c4d993b4e0fec160026fe0ac0017cb7